.travis.yml

```diff
@@ -104,7 +106,6 @@ deploy:
         secure: DdirbOkYUePnKisYPlqYOpyti2HQFWmSIwY5spWflTUglzEZbnp+Q3sEltTebdlWo18i9skz6P5seAAXbX2ye1c3EZhqEyirWMxz1lvGdlNoQGHhWK6x+suM1DrG6Uun5dmcG/CEpWhDpcEbF8xQ/revtyCHEJj4xK9TXUHyFFG/KrzKSLGtsiBG1BOPkHJr4CoHjcpLzpmDB2hn+pYENtl3Nk0EeKmJgWdY/BWctEmUmRAXgfwsO/ejm9IbC5mIy5KBxbIUco/mf2JEKQB1drwdbINMxJXkbx+mWZYiDJBo5CWuWIzNOxv0Z3+Kg+N2cSwPX833v+W5YRTHyef/7Dum1yUrD3T7LMLY+3EIeCoGvARfRnQbi+BM9n1rFobPy/PWHWYxYRDnhDEfXrMcAyH/66InlQIhUh7XD01oZLEbsdZAbGnEsv5Ttz3F9LaEpk3EfoGqLxOWkL50OCxw6Jk6TfxsNtMoLpK1g7Jp8sI/EkjxYISer7S5myzLhKtgbtBwI0RcKh/sUo2m69QX++1JYy6CeN7/EwRmV4zJLk0WgalJuxgNFYrdzPLvg09okgxr1stHZQ7a5RkLH+r8KzA/Lmj6vTQG9HB44lHvX1hMiD0v8aCii+/wfr9AZZlNHjYq8RuHsP5uQ5U5F8G9uMqfiVb75MOPKY7qdYCZZZg=
       file_glob: true
       file: dist/*.*
-      skip-cleanup: true
```

Comment:
Are we sure this isn't needed for deployment? https://docs.travis-ci.com/user/deployment#uploading-files-and-skip_cleanup

Reply:
Pretty sure we do need this!

Reply:
Travis was complaining about that setting being obsolete, since skipping cleanup is now the default, but it could be they've jumped the gun on that and they're still using the older code for now?

Reply:
Ah ok, well we should take them at thier word then I guess.

Reply:
I think that's only the case if we opt-into the new deploy process (which we haven't done here). See https://travis-ci.community/t/skip-cleanup-true-is-now-deprecated-how-to-go-for-gradle-publish/6641/4

Reply:
I've re-added this for now